snap/info.go

```diff
@@ -173,3 +173,25 @@ type AppInfo struct {
 func (app *AppInfo) SecurityTag() string {
 	return fmt.Sprintf("snap.%s.%s", app.Snap.Name(), app.Name)
 }
+
+// WrapperPath returns the path to wrapper invoking the app binary.
+func (app *AppInfo) WrapperPath() string {
+	var binName string
+	if app.Name == app.Snap.Name() {
+		binName = filepath.Base(app.Name)
+	} else {
+		binName = fmt.Sprintf("%s.%s", app.Snap.Name(), filepath.Base(app.Name))
+	}
+
+	return filepath.Join(dirs.SnapBinariesDir, binName)
+}
+
+// ServiceFile returns the systemd service file path for the daemon app.
+func (app *AppInfo) ServiceFile() string {
+	return filepath.Join(dirs.SnapServicesDir, fmt.Sprintf("snap_%s_%s_%d.service", app.Snap.Name(), app.Name, app.Snap.Revision))
```

Comment:
@mvo5 also if I understand these are created on activation, so there's at most of one set per snap name at any moment right? so we don't strictly need revision in there (we could put back version if we want or nothing)